Rx/v2/src/rxcpp/rx-observable.hpp

```diff
@@ -870,6 +870,31 @@ class observable
 
     /*! Determine whether two Observables emit the same sequence of items.
 
+        \tparam OtherSource      the type of the other observable.
+        \tparam BinaryPredicate  the type of the value comparing function. The signature should be equivalent to the following: bool pred(const T1& a, const T2& b);
+
+        \param t     the other Observable that emits items to compare.
+        \param pred  the function that implements comparison of two values.
+
+        \return  Observable that emits true only if both sequences terminate normally after emitting the same sequence of items in the same order; otherwise it will emit false.
+
+        \sample
+        \snippet sequence_equal.cpp sequence_equal sample
+        \snippet output.txt sequence_equal sample
+    */
+    template<class OtherSource, class BinaryPredicate>
+    auto sequence_equal(OtherSource&& t, BinaryPredicate&& pred) const
```

Comment:
on the same topic, the coordination overloads appear to be missing in observable as well.